src/var/clearInterval.js

```diff
@@ -0,0 +1,3 @@
+define(function() {
+	return window.clearInterval;
```

Comment:
> but defining document and documentElement

It's so that we don't need to repeat `window.document` multiple times, risking we'll make a mistake our linter won't catch.